### DISCUSSION TITLE
Where is the list of app platform supported environment slugs?

### DISCUSSION THREAD

**Discussion by ryanvade** [digitalocean/terraform-provider-digitalocean#1190] [General]
I'm trying to create an app via the terraform provider but am running into an issue with unrecognized environment_slug. So, trying to find the supported environment_slug values I ended up on https://docs.digitalocean.com/products/app-platform/reference/app-spec/#example-app-spec

Expanding the spec I end up with 
<img width="930" alt="image" src="https://github.com/user-attachments/assets/ab4b7cbe-c511-47f3-89b1-9d77b6bc30e5">

and clicking on the provided link I end up on the general app product page. I can't find the list of supported environment_slugs. Anyone know where to find them?

**Comment by andrewsomething** [comment]
Hey @ryanvade ,

That's a good question! I asked the App Platform team internally as it appears there is not a publicly documented list of slugs. Environment slugs are being deprecated. You can exclude them completely from your app spec. All that said, currently the valid slugs are:

	"node-js"
	"php"
	"ruby"
	"python"
	"go"
	"hugo"
	"html"
	"hexo"
	"ruby-on-rails"
	"jekyll"
	"gatsby"

**Reply by ryanvade** [reply]
Thank you for the information!
